safe-global/web-core
# fix: add what are signed messages link to page

## What it solves

Resolves #1430 

## How this PR fixes it
Adds a link to the Messages page header if there are off-chain signed messages

## How to test it
1. Go to `Transactions > Messages` in a Safe with off-chain messages signed and you should see the link
2. Go to `Transactions > Messages` in a Safe without off-chain messages signed and you should not see the link above the table but instead in the `NoMessage` placeholder

## Screenshots
<img width="1068" alt="Screenshot 2022-12-27 at 12 17 02" src="https://user-images.githubusercontent.com/32431609/209665560-30219177-0662-4861-8dc1-654e4dbc20c6.png">
<img width="833" alt="Screenshot 2022-12-27 at 12 16 47" src="https://user-images.githubusercontent.com/32431609/209665574-bfdcee22-7b1f-4dce-a091-40645f819da7.png">
